test_scripts/API/Navigation/SendLocation/020_SendLocation_boundary_by_parameters.lua

```diff
@@ -0,0 +1,271 @@
+-- 2) on getting Navigation.SendLocation ("SUCCESS") response from HMI, respond with (resultCode: SUCCESS, success:true)
+-- to mobile application.
+--
+-- Description:
+-- App sends SendLocation request with boundary value of parameters.
+--
+-- Steps:
+-- mobile app requests SendLocation with lower and upper bound value of parameters
+--
+-- Expected:
+-- SDL must:
+-- 1) transfer Navigation.SendLocation to HMI
+-- 2) on getting Navigation.SendLocation ("SUCCESS") response from HMI, respond with (resultCode: SUCCESS, success:true)
+-- to mobile application.
+---------------------------------------------------------------------------------------------------
+
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local commonSendLocation = require('test_scripts/API/Navigation/commonSendLocation')
+local commonFunctions = require('user_modules/shared_testcases/commonFunctions')
+
+--[[ Local Variables ]]
+local maxAdressLinesArraySize = 4
+local string500Char = string.rep("a", 500)
+local string255Char = string.rep("a", 255)
+local string200Char = string.rep("a", 200)
+
+local request_params = {
+    longitudeDegrees = 1.1,
+    latitudeDegrees = 1.1
+}
+
+local addressLinesArrayUpperBoundSize = {}
+for i=1,maxAdressLinesArraySize do
+    addressLinesArrayUpperBoundSize[i] = "some string"
+end
+local addressLinesArrayUpperBoundSizeValue = {}
+for i=1,maxAdressLinesArraySize do
+    addressLinesArrayUpperBoundSizeValue[i] = string500Char
+end
+
+local DateTimeParams =
+{
+    millisecond = {lower = 0, upper = 999},
+    second = {lower = 0, upper = 60},
+    minute = {lower = 0, upper = 59},
+    hour = {lower = 0, upper = 23},
+    day = {lower = 1, upper = 31},
+    month  = {name = "month", lower = 1, upper = 12},
+    --According to comfirmed information before 'should not be any min value specified',
+    -- so as min value specified negative value to check correct processing value not from positive range.
+    year = {lower = -1, upper = 4095},
+    tz_hour = {lower = -12, upper = 14},
+    tz_minute = {lower = 0, upper = 59},
+}
+
+local addressParams =
+{
+    "countryName",
+    "countryCode",
+    "postalCode",
+    "administrativeArea",
+    "subAdministrativeArea",
+    "locality",
+    "subLocality",
+    "thoroughfare",
+    "subThoroughfare",
+}
+
+local boundValues ={
+    longitudeDegrees = {lower = -180, upper = 180},
+    latitudeDegrees = {lower = -90, upper = 90},
+    locationName = {lower = "a", upper = string500Char},
+    locationDescription = {lower = "a", upper = string500Char},
+    phoneNumber = {lower = "a", upper = string500Char},
+    addressLines = {lower = {"a"}, upper = {string500Char}}
+}
+
+local locationImage = {lower = "a", upper = string255Char}
+
+local deliveryModeEnum = {
+    "PROMPT",
+    "DESTINATION",
+    "QUEUE"
+}
+
+--[[ Local Functions ]]
+local function sendLocation(parameter, value, self)
+    local paramters  = commonFunctions:cloneTable(request_params)
```

Comment:
@GetmanetsIrina Please fix typo "paramters"

Reply:
Done in 4875635ea9ce242266c4f0d7bbaece1731a76faf